app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,9 @@
 class WelcomeController < ApplicationController
   def index
   end
+
+  def foods
+    @q = params[:q]
+    @foods = UsdaFoodService.search(@q)
```

Comment:
Light controller! Nice! This should not break our RESTful conventions though, and should be within an `index` action in a `FoodsController`.